deploy/catalog_resources/redhat/activemq-artemis.package.yaml

```diff
@@ -1,5 +1,5 @@
 #! package-manifest: deploy/catalog_resources/redhat/activemq-artemis-operator.v0.4.0.clusterserviceversion.yaml
-packageName: activemq-artemis-operator
+packageName: activemq-artemis
```

Comment:
@redhatHameed Does the package name here generally refer to the distribution that the operator operates? If not I think this may cause some confusion as there are other places where a package name of activemq-artemis would mean the distribution of the broker and not the operator.

Reply:
Yes, this is the package/installation plan file which is required for OLM integration. The OLM configuration is OCP4 is little different vs OCP 3, we need to create an application repository in quay.io with the same name as package name mentioned in this file and push artifacts inside that quay application repository. 

In quay.io, there are two kinds of repository container image and application repositories, as my container image already have the name 'activemq-artemis-operator' as same name not allowed for repo's then I change it as  activemq-artemis for application repository. we can use different name if this is making confusion.

Reply:
@RoddieKieley I would say the package name is mostly internal to OLM. Other than in yaml, I have not seen it exposed anywhere. One important consideration is that it would not clash with the community package name, as both will be in the operator hub some point in the future.